austrakka/components/project/metadata/funcs.py

```diff
@@ -52,29 +53,38 @@ def list_dataset_views(
 def download_dataset_view(
         output_dir: str,
         dataset_view_id: str,
-        abbrev: str):
+        abbrev: str,
+        out_format: str
+):
+    created_dir = False
     if not os.path.exists(output_dir):
         create_dir(output_dir)
+        created_dir = True
 
-    path = "/".join([PROJECT_PATH, abbrev, 'download-project-view', dataset_view_id])
-    _download_dataset_view_file(output_dir, path)
-
-
-def _download_dataset_view_file(dir_path, query_path):
+    query_path = "/".join([PROJECT_PATH, abbrev, 'download-project-view', dataset_view_id])
     try:
-        def _write_chunks(resp: httpx.Response, file):
+        def _write_chunks(resp: httpx.Response):
+            content_disposition = resp.headers.get("Content-Disposition")
```

Comment:
Should make this content disposition parsing into a http util. It's highly reusable.